packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1374,7 +1399,7 @@ abstract class BuildableElement extends Element {
 typedef Widget WidgetBuilder(BuildContext context);
 typedef Widget IndexedBuilder(BuildContext context, int index);
 
-// See _builder.
+// See ComponentElement._builder.
```

Comment:
IDEs can do whatever they want with any comment, with or without markup. I'm reluctant to use `///` style for privates because it gives a confusing mixed message in the code about whether something is considered part of the public interface or not.